### DISCUSSION TITLE
First select delegates and then the tracks

### DISCUSSION THREAD

**Discussion by bkchr** [jeluard/swipealot#455] [Ideas]
Maybe it would make sense to first select the delegates and then the tracks. The advantage would be that the user could delegate all the wanted tracks to all the delegates at once, instead of being required to send multiple transactions.

**Comment by jeluard** [comment]
Overall the idea was to emphasize the `tracks` concept that is the main new angle of OpenGov. 
We might want to explore other selection mechanism that would provide more flexibility indeed.

**Comment by dnlklmn** [comment]
Thanks for the suggestion @bkchr, we wanted to see if a track-first approach solves for _most_ use cases, but I believe a delegate-first flow should be implemented soon (another advantage would be to link to a list of all delegates).

I'm not sure I understand how you could delegate all tracks to all delegates, you would still only be able to pick 1 delegate, assign tracks to them, and repeat

**Reply by bkchr** [reply]
While writing my proposal above, I realized that we probably could also achieve the same with the first select tracks and then delegates flow. In general I would have assumed there is some screen where you maybe can group the tracks and then assign these to a  delegate.
